src/server/session.ts

```diff
@@ -688,9 +689,11 @@ namespace ts.server {
         private suppressDiagnosticEvents?: boolean;
         private eventHandler: ProjectServiceEventHandler | undefined;
         private readonly noGetErrOnBackgroundUpdate?: boolean;
+        private fshost: ServerHost | undefined;

```

Comment:
In webServer/webServer.ts you want to set `LanguageServiceMode.Semantic` mode as allowed and default if vfs is used